en/identity-server/next/docs/references/service-extensions/pre-flow-extensions/pre-issue-access-token-action/api/pre-issue-access-token-action-v1.yaml

```diff
@@ -226,6 +236,25 @@ components:
           example:
             - read
       description: Represents the access token that is about to be issued. It contains claims and scopes, of the access token which can then be modified by your external service based on the logic implemented in the pre-issue access token action.
+    RefreshToken:
+      type: object
+      required:
+        - claims
+      properties:
+        claims:
+          type: array
+          items:
+            $ref: '#/components/schemas/RefreshTokenClaims'
+          description: |
+            An array that contains refresh token claims.
+            
+            Standard claims:
```

Comment:
is having no standard claims listed here expected?

Reply:
We can just refer to expires_in here as that's the only claim rather than making things confusing

Reply:
Basically specify exact claims being returned with refresh token here

Reply:
Fixed fa91ed432a84540f87b805e17e9e977b56cea654